### ISSUE TITLE
Three issues caused by the recent WordPress update

### ISSUE THREAD

**Issue by Unwanted-Life** [Automattic/wp-calypso#51768]
Let me start by saying I have zero technical computer skills, I've tried the WordPress.org forum and WordPress.com for support, but no one seems to be able to help. This seems to be the only place left I can seek help. Also excuse any mistakes with my writing as I'm dyslexic

On my WordPress.com free blog

When you write text, highlight it, go to change colour, select custom colour, then go to move the sliding bar, it won’t slide and picks the colour the bar starts on. But you can see the video from my Google Drive link below

https://drive.google.com/folderview?id=1-OCEMHhRN0E74TuWPV-1BAYbe-GbizJy

Here’s my videos that show all the problems that the update caused on my self-hosted blog which includes the same custom colour problem that’s affecting blogs on WordPress.com and not just self-hosted WordPress blogs. The problem still exists on newly created subdomain’s with default theme and no plugins and in debugging mode

https://drive.google.com/folderview?id=1COk9zIJEwWeQtLX-ZgIBQ5oyWQgtkFL2

Because my WordPress.com blog is a few blog I can't install plugins to test to see if the following issues are also affecting my blog on WordPress.com

The other problems are:

https://wordpress.org/support/topic/latest-update-has-caused-things-to-stop-working/

The added plugin menu options that appear in the sidebar when creating and editing posts and pages don't responded when you click on them.

The problem with my disappearing featured images which started more than a year ago and I tried getting support with it through the WordPress.org forum and from WordPress Beginner (@wpbeginner https://twitter.com/wpbeginner) disappeared with your pervious update but returned with your most recent WordPress update.

https://wordpress.org/support/topic/disappearing-featured-image/

https://wordpress.org/support/topic/featured-image-452/

Even with a completely new, clean, and defaulted WordPress.org blog the issues are there and in debugging mode the issues remain. I’ve accessed both my self-hosted and free WordPress.com blogs on 6 different browsers, two different computers, and cleaned caches galore, and the issues are still there.

**Comment by kwight** [comment]
Hi @Unwanted-Life , I'll try to get you some information here.

> When you write text, highlight it, go to change colour, select custom colour, then go to move the sliding bar, it won’t slide and picks the colour the bar starts on.

This is a [known issue](https://github.com/WordPress/gutenberg/issues/29848#issuecomment-816145103) with Gutenberg (the editor project itself) – I've added your experience to that ticket. Because the problem exists with the editor itself, yes, you will see this issue on WordPress.com (which always runs the latest plugin version of Gutenberg), and as you've seen, any standalone site of the latest WordPress core release. When it's fixed in Gutenberg, it'll be rolled out to WP.com and core for the next release.

I was unable to find references or existing cases of your other two issues (other than the duplicate "[three issues](https://core.trac.wordpress.org/ticket/52994)" ticket you filed). You'll have better luck reporting them as separate, discrete issues to Trac; they'll be easier for core developers to handle, and therefore more likely to get taken care of. Good luck, I hope that helps!

Closing, as this is not specific to WP.com.
